discovery-provider/src/queries/response_name_constants.py

```diff
@@ -46,6 +46,7 @@
 associated_wallets_balance = "associated_wallets_balance"
 associated_sol_wallets_balance = "associated_sol_wallets_balance"
 waudio_balance = "waudio_balance"
+spl_wallet = "spl_wallet"
```

Comment:
do we use this constant anywhere? also missing erc_wallet

Reply:
Used here:
https://github.com/AudiusProject/audius-protocol/pull/2445/files#diff-7a395e2aae36857b886a3b6758451433cdfe7f5df164a369c2503ac99213cbd1R271

We don't use erc_wallet anywhere in the existing response_name_constants, which I kinda dislike, but can add there for consistency